doc/source/conf.py

```diff
@@ -298,24 +295,6 @@
 }
 
 
-# I don't know where and when, but at some point sphinx stopped to detect the documentation
-# hidden below numba. This issue is discussed here https://github.com/sphinx-doc/sphinx/issues/3783
-# pointing to this conf.py:
-# https://github.com/duetosymmetry/qnm/blob/d286cad616a4abe5ff3b4e05adbfb4b0e305583e/docs/conf.py#L71-L93
-# However, it doesn't do the trick truly, but the idea is take from there ...
-# see also
-# https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#docstring-preprocessing
-def process_numba_docstring(
```

Comment:
just for future reference: this PR also drops this hack as it does not seem necessary any longer